README.md

```diff
@@ -456,6 +456,77 @@ q3 = ChemSimilarityQuery(value="InChI=1S/C13H10N2O4/c16-10-6-5-9(11(17)14-10)15-
 list(q3())
 ```
 
+### Faceted Queries
+In order to group and perform calculations and statistics on PDB data by using a simple search query, you can use a faceted query (or facets). Facets arrange search results into categories (buckets) based on the requested field values. More information on Faceted Queries can be found [here](https://search.rcsb.org/#using-facets). All facets should be provided with `name`, `aggregation_type`, and `attribute` values. Depending on the aggregation type, other parameters must also be specified. The `facets()` function runs the query `q` using the specified facet(s), and returns a list of dictionaries:
+```python
+from rcsbsearchapi.search import AttributeQuery, Facet, Range
+
+q = AttributeQuery("rcsb_accession_info.initial_release_date", operator="greater", value="2019-08-20")
+q.facets(facets=Facet("Methods", "terms", "exptl.method"))
```

Comment:
For this and all other uses of `Facet(...)` below, can you please provide the explicit keyword arguments instead of positional arguments?